src/components/BreedCard.vue

```diff
@@ -1,36 +1,17 @@
 <template>
   <li
+    ref="listItemRef"
     @click="toggleDetails"
     :class="!breed.isActive ? 'hidden-result__details' : 'result__list'"
   >
     <div class="result__heading-container">
       <span class="result__id">{{ id + 1 }}.</span>
       <h3 class="heading-tertiary">{{ breed.name }}</h3>
     </div>
-    <div :class="hiddenClass">
-      <div class="result__description">
-        <span class="result__temperament result__item--margin"
-          ><span class="result__description--title">Temperament: </span>
-          {{ breed.temperament }}</span
-        >
-        <span class="result__life result__item--margin"
-          ><span class="result__description--title">Life span: </span>
-          {{ breed.lifeSpan }}</span
-        >
-        <span class="result__height result__item--margin"
-          ><span class="result__description--title">Height: </span>
-          {{ breed.height.metric }}cm</span
-        >
-        <span class="result__weight result__item--margin"
-          ><span class="result__description--title">Weight: </span>
-          {{ breed.weight.metric }}kg</span
-        >
-      </div>
-      <img
-        class="result__breed-image"
-        :src="`https://cdn2.thedogapi.com/images/${breed.reference_image_id}.jpg`"
-        :alt="`Picture of the ${breed.name} dog`"
-      />
+    <div :class="hiddenClass" @click.stop>
```

Comment:
pomoje še vedno lahko prikažeš ime breeda in sliko v kartici.. več info pa pol na details pageu, kaj misleš? da ne bo samo gumb

Reply:
Se strinjam, da ne bo tok prazno :)

Reply:
Sem dodal sliko pasme, če klikneš na kartico. Sem jo pa dodal kar poleg imena pasme, ker se mi zdi boljse kako pa spodaj pri gumbu, kaj mislis?